src/applications/personalization/rated-disabilities/components/RatedDisabilityList.jsx

```diff
@@ -56,11 +56,16 @@ class RatedDisabilityList extends React.Component {
   formalizeData(data) {
     const formalizedDisabilityData = _.map(data, d => {
```

Comment:
Is `data` an object or array? If it's an array I'd prefer using the native map()